test/runtests.jl

```diff
@@ -1,18 +1,14 @@
 using SpinGlassNetworks
+using LabelledGraphs
 using LightGraphs
 using MetaGraphs
 using Logging
-
-disable_logging(LogLevel(1))
-
 using Test
 
-my_tests = []
-push!(my_tests,
-      "ising.jl",
-      "factor.jl",
-)
+Base.:(==)(e1::LabelledEdge, e2::LabelledEdge) = src(e1) == src(e2) && dst(e1) == dst(e2)
```

Comment:
Please move to `LabelledGraphs`

Reply:
Yes, I will move this next week. The new version of LabelledGraphs will also contain the `add_edge!` method that accepts a dictionary of properties (in this PR we temporarily changed it to `add_edge!` followed by `set_props`).